presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveViews.java

```diff
@@ -172,19 +169,24 @@ public void testRichSqlSyntax()
                 "   CASE WHEN n_name = \"BRAZIL\" THEN 'is BRAZIL' WHEN n_name = \"ALGERIA\" THEN 'is ALGERIA' ELSE \"\" END is_something,\n" + // searched CASE, double quote string literals
                 "   COALESCE(IF(n_name LIKE 'A%', NULL, n_name), 'A%') AS coalesced_name, \n" + // coalesce
                 "   round(tan(n_nationkey), 3) AS rounded_tan, \n" + // functions
+                "   o_orderdate AS the_orderdate, \n" +
                 "   `n`.`n_nationkey` + `n_nationkey` + n.n_nationkey + n_nationkey + 10000 - -1 AS arithmetic--some comment without leading space \n" +
-                "FROM `default`.`nation` AS `n`");
+                "FROM `default`.`nation` AS `n` \n" +
+                // join, subquery
+                "LEFT JOIN (SELECT * FROM orders WHERE o_custkey > 1000) `o` ON `o`.`o_orderkey` = `n`.`n_nationkey` ");
```

Comment:
same comment as previously, it would be nice to have single test per case

Reply:
You can give it a try. I did not because i strngly believe this would take prohibitively long time